axelrod/tests/unit/test_player.py

```diff
@@ -20,7 +20,9 @@ class TestPlayerClass(unittest.TestCase):
 
     name = "Player"
```

Comment:
but all the tests are also inherited - from TestPlayer. Anything you put in its ```test_initialisation``` method will also be used by all the tests for all the strategies.